redis-sink/src/main/resources/application.yml

```diff
@@ -0,0 +1,12 @@
+server:
+  port: 8081
+spring:
+  cloud:
+    stream:
+      bindings:
+        input: testtock
```

Comment:
The trouble with using a single value across the board is that it works very well with sources and sinks, but falls apart with processors (which can't use the same value for input and output).